IFC4x3/PropertyConstants/t/TRACKPANEL_0IocCGCbjEbQgMuEL_GjIF.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TRACKPANEL_0IocCGCbjEbQgMuEL_GjIF" Name="TRACKPANEL" UniqueId="12ca6310-325b-4e95-aa96-e0e57e42d48f" />
+
```

Comment:
No documentation, even if it is maybe not entirely necessary here.

Reply:
erased. Comming from Element assembly. Commit 6cb18533994026ae118add924a2b65d2f98b8472